app/models/company.rb

```diff
@@ -0,0 +1,2 @@
+class Company < ApplicationRecord
```

Comment:
do we need:
has_many: users
has_many: set_ups

Reply:
Okay, will add now